README.md

```diff
@@ -55,25 +65,38 @@ DynatraceMetricExporter exporter =
 After acquiring a `DynatraceMetricExporter` object, it has to be registered with the OpenTelemetry SDK using a `MetricReader`:
 
 ```java
+import java.time.Duration;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
```

Comment:
I'm unsure about including the imports, but leaving out the class and method. It may be confusing, as it gives the impression that it's copy-pasteable in its entriety. Not sure if this is just me though :thinking:

Reply:
Yeah, if using an IDE it's not a problem. I added it mostly because a little bit down there's another snippet that had it so I thought of making it consistent at least. If we think it's not necessary I will remove from all of them.

Reply:
I tend towards not having them. I assume most users will use an IDE or they know what they are doing anyway.

Reply:
OK, remove it is then :)